static/scss/profile.scss

```diff
@@ -32,6 +32,9 @@
     flex-direction: row;
     color: $lightgrey !important;
     text-transform: uppercase;
+    position: absolute;
+    right: 0;
+    padding-right: 50px;
```

Comment:
if you get rid of these three lines the layout seems to be the same.

with:

![with_lines](https://cloud.githubusercontent.com/assets/6207644/18315327/99628e34-74e4-11e6-9acd-b89ff6f512ba.png)

without:

![without_lines](https://cloud.githubusercontent.com/assets/6207644/18315338/9f3ac63c-74e4-11e6-9058-7eaa3f38f8b0.png)

let's delete them